src/main/kotlin/com/jfrog/conan/clionplugin/services/RemotesDataStateService.kt

```diff
@@ -41,12 +61,15 @@ class RemotesDataStateService : PersistentStateComponent<RemotesDataStateService
          * @param e a `TableModelEvent` to notify listener that a table model
          * has changed
          */
-        fun stateChanged(newState: State)
+        fun stateChanged(newState: State?)
     }
 
 
     fun addStateChangeListener(listener: RemoteDataStateListener) {
         listeners.add(RemoteDataStateListener::class.java, listener)
+        if (state != null) {
+            listener.stateChanged(state)
+        }
```

Comment:
This if is actually wrong. sThe compiler complains that as state is mutable, checking its nullidity and then calling it is not correct, it could have become null by that point. I didn't quite know how to handle that so instead the listener now accepts a null state - Would be nice to go back to the other way in which that's not possible